pkg/log/log.go

```diff
@@ -30,3 +30,17 @@ type Logger interface {
 	// Failuref logs a formatted failure message.
 	Failuref(format string, a ...interface{})
 }
+
+type DiscardLogger struct{}
```

Comment:
Please move this to a new file e.g. `nop.go` and rename the type to `NopLogger` like zap does.

Reply:
Done

Reply:
Wouldn't simply `log.Discard` make more sense?

Reply:
My above comment wasn't taken into account when this was merged.